content/components/selectpanel.mdx

```diff
@@ -9,51 +9,155 @@ import {Box} from '@primer/react'
 
-### Additional control slot
+### Trigger
+
+<img width="960" alt="" src="https://github.com/primer/design/assets/980622/f6a1e670-2682-4ce7-8642-24943084c812" />
+
+Select panels can be activated by either a regular [button](/components/button), [icon button](/components/icon-button) or [action menu](/components/action-menu). If the button is intended to represent the current selection, it's crucial to have a label associated with it, either internally (within the button) or externally (adjacent to the button).
+
+### List (single select)
 
 <img
   width="960"
-  alt="Three cropped select panel footers each displaying a different option of a control in the bottom right left corner: Link, button, or checkbox input."
-  src="https://user-images.githubusercontent.com/40274682/229011396-2d2c6132-937b-49dd-974c-1a8d4d0d4737.png"
+  alt="Multi-select"
+  src="https://github.com/primer/design/assets/980622/3d0cc2b5-b958-4b62-805e-f77a9d7bf0d9"
 />
-<Caption>An optional control slot is available in the select panel’s bottom left corner. Avoid use if not absolutely necessary in order to avoid component and decision overwhelm.</Caption>
 
-### Anchoring
+If the select panel doesn't have an additional checkbox in the footer, you can make an instant selection. This means that the select panel will close as soon as you make the selection.
 
-Select panel should be anchored to its trigger. 
+However, if there is an additional form element required for that selection, such as checking an additional option, it is necessary to have a cancel and confirmation button. This prevents the dialog from unexpectedly closing when you click an option, allowing you to review and confirm your selection.
 
-<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
-  <div>
-    <img
-      src="https://user-images.githubusercontent.com/40274682/229016799-9559a4ee-0daf-42d1-b46d-11c64cf182e4.png"
-      alt=""
-    />
-    <Caption>Anchored select panel to a trigger.</Caption>
-  </div>
-</Box>
+To indicate an instant selection, we use checkmarks to show the selected option. For regular single selects, we use visually styled radio buttons. Note they are not semantically radio buttons.
+
+### List (multi select)
+
+<img
+  width="960"
+  alt="Multi-select"
+  src="https://github.com/primer/design/assets/980622/33176d07-2853-4318-bf5d-df395e700fa9"
+/>
+
+The multi-select list allows you to choose multiple items from a list and has additional features like `select all` or `indeterminate selections`.
+
+The `select all` option at the top of the list allows for quickly deselecting or selecting all values.
+However, when you separate items in groups this option isn't available anymore.
```

Comment:
I'm not sure what you mean by "separate items in groups"

Reply:
Updated this to be more clear based on the API exploration I did:
`However, when implementing search functionality, this option becomes unavailable.`